### ISSUE TITLE
In MS Word, NVDA does not annunce when text is unselected

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#10518]
### Steps to reproduce:

* Open MS Word
* Type some text
* Select some text
* Press left or right arrow key to cancel selection

### Actual behavior:

NVDA announce the character where the cursor has been moved.
Text is unselected but NVDA did not announce that text has been unselected. 

### Expected behavior:

NVDA announce the character where the cursor has been moved and anounce that text has been unselected.
This way, Word's text unselection would be conform to what happens in Notepad or Wordpad.

An other option would be that neither Word, nor Wordpad, nor Notepad announce that text has been unselected.

### Additional information

I do not feel personally affected by this issue. I have opened it on @Adriani90's suggestion to track an off-topic discussion in #1898.

#### NVDA installed/portable/running from source:

Installed

#### NVDA version:

2019.2.1

#### Windows version:

Windows 10

#### Name and version of other software in use when reproducing the issue:

N/A

#### Other information about your system:

N/A

#### Does the issue still occur after restarting your PC?

Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.

No

**Comment by XLTechie** [comment]
To the question: Have you tried any other versions of NVDA? If so, please report 
their behaviors: in the other issue, I did report trying this and getting the 
same results in 2019.3 alpha.

**Comment by CyrilleB79** [comment]
OK. I have tested also with last alpha run from source (source-master-833bc4e) and NVDA 2017.3 portable.
The issue has always been reproduced whatever the version.

**Comment by leonardder** [comment]
Is this with UIA enabled or disabled? Does that make a difference?

Please also provide the Word version in use.

**Comment by CyrilleB79** [comment]
All these tests were made with default option, i.e. UIA disabled.

On NVDA 2019.2.1, enabling UIA does not make a difference: issue still present.

On NVDA 2019.2.1 and probably others, if I activate browse mode (Ctrl+Space), NVDA announces when the text is unselected.

**Comment by CyrilleB79** [comment]
This issue seems to be fixed from NVDA 2019.3.1 onward, since "selection removed" / "unselected" is not reported anymore upon cursor move, neither in MS Word, nor in Notepad, nor in WordPad.
Note: I do not know what the reported message was ("unselected" or "selection removed").

The reported messages are now coherent among these various applications.
The following expected result has been achieved:
> An other option would be that neither Word, nor Wordpad, nor Notepad announce that text has been unselected.

@Adriani90, @XLTechie and @ehollig, since you commented quite recently here or in #1898, could you confirm there is no more issue for you and this way to do is OK? In this case, I'll close this issue.

**Comment by Adriani90** [comment]
This works only in browse mode, also in MS Word. It's the same behavior like in Browsers.
The difference is that in browsers "not selected" is also reported in focus mode.
cc: @JulienCochuyt do you think it makes sense to implement this also for editors? I think this information needs to be exposed some how at system carret. right?

**Comment by CyrilleB79** [comment]
I cannot reproduce the coherent result I have presented in my last comment. I am not on the same PC. Strange...

**Comment by Adriani90** [comment]
When UIA is enabled for MS Word in NVDA advanced settings, this works as expected. NVDA announces when unselecting text in focus mode as well, same as in browse mode.
